src/common/IOSUtils.ts

```diff
@@ -158,20 +159,26 @@ export class IOSUtils {
 
     public static async getSupportedRuntimes(): Promise<string[]> {
         const configuredRuntimes = await IOSUtils.getSimulatorRuntimes();
-        const supportedRuntimes: string[] = iOSConfig.supportedRuntimes;
+        const minSupportedIOSRuntime: Version = new Version(
+            iOSConfig.minSupportedIOSRuntime
+        );
         const rtIntersection = configuredRuntimes.filter(
             (configuredRuntime) => {
-                const responsiveRuntime = supportedRuntimes.find(
-                    (supportedRuntime) =>
-                        configuredRuntime.startsWith(supportedRuntime)
+                const configuredRuntimeVersion = new Version(
+                    configuredRuntime.toLowerCase().replace('ios-', '')
```

Comment:
Technically, this isn't wholly accurate—and could be problematic—because `IOSUtils.getSimulatorRuntimes()` actually [pulls back `iOS`, `watchOS`, and `tvOS`](https://github.com/forcedotcom/lwc-dev-mobile/blob/21360190c4f6fdcbea505f1a12a3e97498a95e4a/src/common/IOSUtils.ts#L179) (meant to be forward-looking/extensible).

That said, I'm in favor of culling that platform list to iOS. I don't know of the world where we're going to end up supporting watchOS or tvOS, and we can always add them back in if such a day comes. And if we cull the list, this logic will be fine.

Reply:
Yes I knew that, but I also thought that we're probably not going to support anything other than iOS anytime soon.... and if we ever do, then we can add the support back since it's easy.